dts/bindings/iio/afe/current-sense-amplifier.yaml

```diff
@@ -45,3 +45,10 @@ properties:
 
       If present the corresponding GPIO must be set to an active level
       to enable the current amplifier.
+
+  noise-threshold:
```

Comment:
since this is a software behavior configuration, needs to be prefaced with `zephyr,`

Reply:
@rruuaanng please help monitor this rule